style.css

```diff
@@ -1 +1,165 @@
-/*  Add your styling here */
+*{
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    -webkit-font-smoothing: antialiased;
+    background-color: white;
+}
+
+
+@media screen and (max-width: 480px) {
+
+.top-bar {
+    background-color: #ffc0cb;
+    width: 100%;
+    padding: 10px;
+    border-bottom: 1px solid black;
+}
+
+.top-bar h1 {
+    text-align: center;
+}
+
+
+.hamburger-menu {
+        display: block;
+        height: 100px;
+        width: 100%;
+        background: url(images/hamburger-menu.png) no-repeat 98% center;
+        background-color: white;
+        padding-left: 35px;
+
+}
+
+.logo {
+    width: 100px;
+    float: left;
+}
+
+#nav-bar {
+    text-align: left;
+    padding-top: 65px;
+}
+
+#nav-bar ul {
+    list-style: none;
+    margin: none;
+}
+
+
+#nav-bar li {
+    display: inline-flex;
+}
+
+#nav-bar li a {
+    color: black;
+    text-decoration: none;
+    padding: 10px;
+
+}
+
+#nav-bar li a:hover {
+    color: black;
+}
+
+.img-banner {
+    display: none;
+}
+
+
+.mini-banner {
+    background-color: #ffc0cb;
+}
+
+.mini-banner h1 {
+    text-align: center;
+    font-size: 35px;
+    padding-top: 60px;
+    padding-bottom: 10px;
+}
+
+.mini-banner p {
+    text-align: center;
+    font-size: 35px;
+    padding-bottom: 60px;
+    margin-bottom: 50px;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.btn {
+    justify-content: center;
```

Comment:
for this to take effect, you also need to make this a flex container, so `display: flex`